modules/registry-accessing-metrics.adoc

```diff
@@ -18,12 +18,47 @@ endpoint.
 
 .Procedure
 
-There are two ways in which you can access the metrics, running a metrics query
-or using the cluster role.
+There are two ways in which you can access the metrics, using the cluster role or running a metrics query.
```

Comment:
Need update this description,
How about 
```There are a way in which you can access the metrics, running a metrics query using the cluster role``` ?

Reply:
How about "To access the metrics, run a metrics query or use the cluster role."
or "You can access the metrics by running a metrics query or using a cluster role."